src/easyscore.ts

```diff
@@ -57,10 +57,11 @@ export class EasyScoreGrammar implements Grammar {
       expect: [this.DURATION, this.TYPE, this.DOTS, this.OPTS],
     };
   }
-  CHORDORNOTE(): Rule {
+  CHORDORNOTEORGHOST(): Rule {
     return {
       expect: [this.CHORD, this.SINGLENOTE],
       or: true,
+      maybe: true,
```

Comment:
What's the intent of this? I think this will accept a lot of invalid syntax.

Reply:
This is the GhostNote. GhostNotes can be now described as `/q/g` they do not need a pitch.

Reply:
This breaks the grammar in a messy way, so I'm not really a fan of it. I'd rather just keep the pitch (despite it not needing it.)

Reply:
Done